core/src/test/java/org/elasticsearch/search/fields/SearchFieldsIT.java

```diff
@@ -872,7 +866,7 @@ public void testScriptFields() throws Exception {
 
     public void testLoadMetadata() throws Exception {
         assertAcked(prepareCreate("test")
-                .setSettings("index.mapping.single_type", false)
+                .setSettings("index.version.created", Version.V_5_6_0.id)
```

Comment:
If we have another test for routing I'd rename this one to `testLoadParent` or something. Then we can nuke it in 7.0 easilly.